website/source/docs/configuration/storage/zookeeper.html.md

```diff
@@ -66,16 +66,11 @@ znodes and, potentially, take Vault out of service.
     ip:70.95.0.0/16
     ```
 
-- `auth_info` `(string: "")` – Specifies an authentication string in Zookeeper
```

Comment:
auth_info is removed in this commit since it was incorrectly committed as a duplicate in a previous commit.